pattern_importer.py

```
@@ -0,0 +1,322 @@
+                regex_match = re.search(r'regex\s*=\s*\'\'\'([^\']+)\'\'\'', section, re.DOTALL)
+                if not regex_match:
+                    regex_match = re.search(r'regex\s*=\s*"([^"]+)"', section)
+                
+                if regex_match:
+                    regex_pattern = regex_match.group(1).strip()
+                    patterns[rule_id] = regex_pattern
+            
+            logger.info(f"Loaded {len(patterns)} patterns from GitLeaks")
+            return patterns
+            
+        except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)